src/components/StatusButton/__test__/index.test.jsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import StatusButton from "..";
+
+const MockStatusButton = () => {
```

Comment:
????

Reply:
I used for this function for mocking a render with props